composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
@mibstar my suggestion is that we don't implement it. We only explain in the documentation how other people can implement it. That means that we don't change the code, only documentation.